vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionAboutPage.java

```diff
@@ -32,6 +37,16 @@ public class ExhibitionAboutPage {
     
     @Lob
     private String aboutPageText;
+    
+    
+    
+    @OneToMany(targetEntity = LocalizedText.class, cascade={CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name="AboutPage_LocText_titles")
+    private List<ILocalizedText> exhibitionTitles = new ArrayList<ILocalizedText>();
+
+    @OneToMany(targetEntity = LocalizedText.class, cascade={CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name="AboutPage_LocText_descriptions")
+    private List<ILocalizedText> exhibitionTextDescriptions = new ArrayList<ILocalizedText>();
```

Comment:
same as above, why is is called `exhibitionTextDescriptions`? It holds the localized about text, doesn't it?